src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from "react";
+import "./navbar.scss";
+
+class Navbar extends Component {
+    render(){
```

Comment:
De acordo com o [style guide do airbnb](https://github.com/airbnb/javascript#functions--signature-spacing), adicionar o seguinte espaçamento melhora a legibilidade:

```suggestion
    render () {
```